### PR TITLE
added etherman to dapp list

### PR DESCRIPTION
**Pull request by mandrigin**
fixed version of https://github.com/status-im/status-react/pull/7552

status: ready

### CODE REVIEW

File: src/status_im/react_native/resources.cljs [hunk 1]
```diff
@@ -36,6 +36,7 @@
    :blockimmo          (js/require "./resources/images/contacts/blockimmo.png")
    :cryptocribs        (js/require "./resources/images/contacts/cryptocribs.png")
    :cryptofighters     (js/require "./resources/images/contacts/cryptofighters.png")
+   :etherman 	       (js/require "./resources/images/contacts/etherman.png")
```
Comment (flexsurfer) [inline src/status_im/react_native/resources.cljs]: one perfectionist cries somewhere ;)

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/react_native/resources.cljs (hunk 1 above)
Comment (mandrigin) [reply src/status_im/react_native/resources.cljs]: oh, lol, didn't notice that :-)

**Review by rachelhamlin — APPROVED** [review]

**Comment by rachelhamlin** [comment]
Thanks @mandrigin!

**Review by goranjovic — APPROVED** [review]

**Review by andytudhope — APPROVED** [review]

**Comment by churik** [comment]
@mandrigin @rachelhamlin 
Indeed, `etherman` is shown of the list of Dapps now.

But I can't proceed with this game becuase button 'Submit' does nothing for me - tried on both Android and IOS.
Also on IOS can't zoom out screen.
If I try to open app again - `Authorization failed` popup is shown, so it is the dead end for my account and this DApp.
My experience with this DAppis here: https://take.ms/gduv1

So the question is - should it be presented in this case in DApps list or not?

<blockquote><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/ZzBl3WwIQIvobO6ynZnMil1st382cr">File "screencast 2019-02-27 12-05-53.mp4"</a></strong></div><div>Monosnap — the best tool to take & share your screenshots.</div></blockquote>

**Comment by rachelhamlin** [comment]
Nope, hold off. Thanks for testing properly @churik. 

@ether-man have you tested inside Status? The DApp doesn't work with privacy mode on—probably not compliant with [EIP1102](https://status.im/developer_tools/access_eth_provider.html).

**Comment by ether-man** [comment]
> Nope, hold off. Thanks for testing properly @churik.
> 
> @ether-man have you tested inside Status? The DApp doesn't work with privacy mode on—probably not compliant with [EIP1102](https://status.im/developer_tools/access_eth_provider.html).

Yes i tested the Dapp inside status. We made some changes today, maybe you have to clear your cache. With privacy mode on you can not play of course.

**Comment by jeluard** [comment]
> With privacy mode on you can not play of course.

What do you mean @ether-man ? Privacy mode will be the only option in the future.

**Comment by ether-man** [comment]
Hi Rachel,

sorry for late reply. We fixed our problem with the input field for the
username.
Would be great if you could give it a second try. This is a really fun game
and as far as i know the only multiplayer dapp!


Regarding the privacy mode: there must be some missunderstanding from my
side:
I understood it like this: if privacy mode is enabled there is no access to
the users wallet from within the dapp.
How can i trigger a transaction in this case?
Cant find any guide on status.im

Thanks
stefan




Am Mi., 13. März 2019 um 16:41 Uhr schrieb Julien Eluard <
notifications@github.com>:

**Comment by jeluard** [comment]
@ether-man Actually it's a new API you will have to use to access the ethereum node. You can read more about it [here](https://our.status.im/0-9-33/)

**Comment by jeluard** [comment]
@churik Maybe you can give it another try now?

**Comment by ether-man** [comment]
cant find any detailed instructions in your link.

we adapted already metamask's considerations:

        if (window.ethereum) {
            window.ethereum.enable().then(function(accounts) {
                window.web3 = new Web3(window.ethereum);
                 ....
}








Am Di., 19. März 2019 um 17:30 Uhr schrieb Julien Eluard <
notifications@github.com>:

**Comment by churik** [comment]
I retested build with https://github.com/status-im/status-react/commit/24eedaeeacfe04f251c99083b75ccf345eca6677  on IOS and Android.
I can log in and host a game (required transaction of 0.001 ETH) normally (browser privacy mode is off). So https://github.com/status-im/status-react/pull/7591#issuecomment-467825805 is fixed.

I can't do anything with enabled browser privacy mode, but it's not only one Dapp with similar behavior, so I see no reason from blocking Etherman from DApp Store.

cc @rachelhamlin @ether-man

**Comment by rachelhamlin** [comment]
Thanks @ether-man for the fix and @churik for testing!

Stefan, you can read our guidelines to EIP1102 here: https://status.im/developer_tools/access_eth_provider.html

I believe it works the same way in any browser, so if you've followed Metamask's code example, should be okay with ours as well (right @jeluard?).

**Comment by jeluard** [comment]
Definitely! You should be good @ether-man

**Comment by ether-man** [comment]
Are you always approving dapps without even testing them completely?
Does the Status team not enjoy playing a game once in a while?

Am Do., 21. März 2019 um 17:20 Uhr schrieb Julien Eluard <
notifications@github.com>:

**Pull request merged**